app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -328,6 +328,8 @@ export class GraphDb {
     moduleCode: string,
     getSpan: (id: AstId) => SourceRange | undefined,
     dirtyNodes: Set<AstId>,
+    onNodeAdded: (id: NodeId) => void,
+    onNodeDeleted: (id: NodeId) => void,
```

Comment:
These are redundant to ReactiveDB's `entryAdded` event (which is available in public)